bindings/python/opendal.pyi

```diff
@@ -11,4 +11,5 @@ class AsyncOperator:
     async def stat(self, path: str) -> Metadata: ...
 
 class Metadata:
+    @property
```

Comment:
In the future, we may consider returning `Option<String>` for other fields. Can `@property` accommodate this?

Reply:
I think it's fine, it can be changed to in that case

```python
@property
def foo(self) -> str | None: ...
```

Reply:
yes, property is the same as a method that takes no argument

Reply:
Oh yeah, I have received guidance from many awesome Python teachers.

Today's lessons have been learned. Have a pleasant evening, teachers!